packages/components/socialFeed/NewsFeed/NewsFeed.type.ts

```diff
@@ -44,6 +48,22 @@ export interface SocialFeedMetadata {
   // openGraph?: OpenGraphType;
 }
 
+export const ZodSocialFeedArticleMetadata = z.object({
+  title: z.string(),
+  shortDescription: z.string(),
```

Comment:
this should be optional since we don't have it in previous posts I think

Reply:
If no `shortDescription`, the old way is used (Truncate the `message` html and show few lines of that)
See https://github.com/TERITORI/teritori-dapp/blob/9bda66ee2d4e8cfcb99b6a009e5e2be8e9ce3f67/packages/components/socialFeed/SocialCard/cards/SocialArticleCard.tsx#L95